android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativePackageImpl.kt

```diff
@@ -20,6 +22,27 @@ import io.customer.reactnative.sdk.util.assertNotNull
  * and their corresponding factory methods.
  */
 internal object CustomerIOReactNativePackageImpl {
+    private const val TAG = "[CIO]"
+
+    /**
+     * Modules that use React Native EventEmitter and crash on armeabi-v7a due to C++ TurboModule issues.
+     * These modules will be skipped on armeabi/armeabi-v7a architectures.
+     */
+    private val EVENT_EMITTER_MODULES = setOf(
+        NativeCustomerIOLoggingModuleImpl.NAME,
+        NativeMessagingInAppModuleImpl.NAME
+    )
```

Comment:
I understand the challenge of not being able to test this and the intent to avoid invoking those classes. Suppressing the logger module makes sense since it won't affect production much. But disabling in-app module would break inline messages as they depend on it for sizing callbacks.

However, we could still consider it if we clearly call it out as a limitation until we find a reliable fix. So customers can still use other features without unexpected crashes.

Also, if we haven't already, we could temporarily tweak the ABI check on any device to test the experience with these modules disabled.